### DISCUSSION TITLE
DIY Chem Controller Reusing Intellichem Hardware

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#707] [REM] [answered]
My touch panel on my Intellichem no longer works.  I figure now is the time to get an RPI, relay board, and REM software up and going.

I bought an enclosure for the hardware last year, but the Intellichem is working except for the touchpanel.  So, maybe I can reuse the power supply for running the dosing pumps in addition to the flow cell and ORP/PH probes.

Anyone already "been there, done that"?  Any advice?

Thanks!

**Comment by tagyoureit** [comment, marked as answer]
Posted this in your other question, but check out the wiki which is full of great documentation.  Many of us are running REM chem in some form.  You may be able to reuse a lot, or everything.  I'm not recalling exactly how Intellichem interfaces with their ORP/PH probes, but you should be able to tap into the dosing pumps pretty easily.

[REM Chem wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-REM-Chem-Controller)
